### DISCUSSION TITLE
app opens twice

### DISCUSSION THREAD

**Discussion by name392** [The-Wineskin-Project/WineskinServer#224] [Q&A]
Hi,

In screen options, if i set mac driver, and then scale up fonts in advanced/tools/config utility /graphics/screen resolution  they end up returning to 96 dpi sooner or later, don´t stay at the value i choose.

Tried  x11 driver advanced/tools/config utility /graphics/screen resolution , changing 96 dpi to 130 dpi, works. There´s just one little annoying problem, apps sometimes (luckily not always) open twice with just one click on app shortcut. It´s not a big problem, a bug?
thanks.

**Comment by name392** [comment]
edit..

**Reply by Gcenx** [reply]
I’d quickly checked this with the offline head and it doesn’t seem to be happening so maybe I’d already resolved this, so the next wrapper bump _should_ have resolved this issue

**Reply by name392** [reply]
It seems that  increasing the fonts from 96 dpi to 120, 125, 128, etc.. in utility /graphics/screen resolution is working when choosing mac driver. I think what was messing with it  was that i was changing the colors in theme options\config utility to mimic osx dark theme and fonts wouldn´t "stay" at the value i chose (returning after some time to the default 96 dpi), decided to try  installing a windows theme,then increase font size.. and it´s working now!

When choosing the x11 driver the problem of apps loading 2 times with one click still happens  (im using mojave) but now i don´t need to choose the x11 driver anymore, so problem solved! thanks!
